subprojects/gradle/cd/src/main/kotlin/com/avito/ci/steps/LintCheck.kt

```diff
@@ -28,6 +29,10 @@ class LintCheck(context: String, name: String) : SuppressibleBuildStep(context,
 
         if (useImpactAnalysis && !project.internalModule.isModified()) return
 
+        rootTask.dependsOn(
+            AndroidLintAccessor(project).taskProvider()
+        )
```

Comment:
With the broken `lint-report` plugin we were not able to run Lint in PR.
Made sending results optional for graceful degradation.